src/main/java/hng_java_boilerplate/user/serviceImpl/UserServiceImpl.java

```diff
@@ -63,6 +66,8 @@ public ResponseEntity<ApiResponse> registerUser(SignupDto signupDto) {
         validateEmail(signupDto.getEmail());
 
         User user = new User();
+        Plan plan = planService.findOne("1");
```

Comment:
Did you need to add this? the plan can be assigned to the user when a user wants to set plan